Source/Relativity.Test.Helpers/Relativity.Test.Helpers.NUnit.Integration/test.runsettings

```diff
@@ -0,0 +1,21 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<RunSettings>
+	<!-- Parameters used by tests at run time -->
+	<TestRunParameters>
+		<Parameter name="WorkspaceID" value="" />

```

Comment:
Instead of empty, can we put some dummy values in there. I think it will give more clarity for users when updating the values. Right now when they are blank, I don't know what the format the value should be in.

Reply:
Updated, please review